app/models/nufs_account.rb

```diff
@@ -1,5 +1,6 @@
 class NufsAccount < Account
-  validates_uniqueness_of :account_number, :message => "already exists"
+  validates_uniqueness_of :account_number, message: "already exists"
+  validates_presence_of :expires_at
```

Comment:
What prompted adding this validation check? Looks to be always be the case, but just curious the impetus.